### ISSUE TITLE
Make the server push/pull rules window resizeable

### ISSUE THREAD

**Issue by cudeso** [MISP/MISP#5518]
**Describe the solution you'd like**
Make the window to add push/pull rules resizeable. It's difficult to differentiate between tags starting with the same text/string.

**Additional context**
![2020-01-16 10_59_38-Servers - MISP](https://user-images.githubusercontent.com/256028/72515144-a8930280-384f-11ea-818e-1e325560a50b.png)

**Comment by iglocska** [comment]
Yeah we need to just scrap that interface, it's ridiculous. Make sure to point and laugh at the monkey that thought that was a good idea.
